contracts/epochs/FlowEpoch.cdc

```diff
@@ -795,6 +805,12 @@ pub contract FlowEpoch {
         return self.account.copy<Bool>(from: /storage/flowAutomaticRewardsEnabled) ?? false
     }
 
+    /// Gets the current amount of bonus tokens left to be destroyed
```

Comment:
I think a more detailed description might be nice here

Reply:
added